docs/pin.rst

```diff
@@ -23,57 +24,65 @@ depending upon the digital reading on pin 0::
-+-----+---------+----------+
-|  3  | Analog  | Column 1 |
-+-----+---------+----------+
-|  4  | Analog  | Column 2 |
-+-----+---------+----------+
-|  5  | Digital | Button A |
-+-----+---------+----------+
-|  6  | Digital | Column 9 |
-+-----+---------+----------+
-|  7  | Digital | Column 8 |
-+-----+---------+----------+
-|  8  | Digital |          |
-+-----+---------+----------+
-|  9  | Digital | Column 7 |
-+-----+---------+----------+
-|  10 | Analog  | Column 3 |
-+-----+---------+----------+
-|  11 | Digital | Button B |
-+-----+---------+----------+
-|  12 | Digital |          |
-+-----+---------+----------+
-|  13 | Digital | SPI SCK  |
-+-----+---------+----------+
-|  14 | Digital | SPI MISO |
-+-----+---------+----------+
-|  15 | Digital | SPI MOSI |
-+-----+---------+----------+
-|  16 | Digital |          |
-+-----+---------+----------+
-+-----+---------+----------+
-|  19 | Digital | I2C SCL  |
-+-----+---------+----------+
-|  20 | Digital | I2C SDA  |
-+-----+---------+----------+
++-----+---------+----------+-----------------+
+| Pin | Type    | Function | Function **V2** |
++=====+=========+==========+=================+
+|  0  | Touch   | Pad 0    | Pad 0           |
++-----+---------+----------+-----------------+
+|  1  | Touch   | Pad 1    | Pad 1           |
++-----+---------+----------+-----------------+
+|  2  | Touch   | Pad 2    | Pad 2           |
++-----+---------+----------+-----------------+
+|  3  | Analog  | Column 1 | Column 3        |
++-----+---------+----------+-----------------+
+|  4  | Analog  | Column 2 | Column 1        |
++-----+---------+----------+-----------------+
+|  5  | Digital | Button A | Button A        |
++-----+---------+----------+-----------------+
+|  6  | Digital | Column 9 | Column 4        |
++-----+---------+----------+-----------------+
+|  7  | Digital | Column 8 | Column 2        |
++-----+---------+----------+-----------------+
+|  8  | Digital |          |                 |
++-----+---------+----------+-----------------+
+|  9  | Digital | Column 7 |                 |
++-----+---------+----------+-----------------+
+|  10 | Analog  | Column 3 | Column 5        |
++-----+---------+----------+-----------------+
+|  11 | Digital | Button B | Button B        |
++-----+---------+----------+-----------------+
+|  12 | Digital |          |                 |
++-----+---------+----------+-----------------+
+|  13 | Digital | SPI SCK  | SPI SCK         |
++-----+---------+----------+-----------------+
+|  14 | Digital | SPI MISO | SPI MISO        |
++-----+---------+----------+-----------------+
+|  15 | Digital | SPI MOSI | SPI MOSI        |
++-----+---------+----------+-----------------+
+|  16 | Digital |          |                 |
++-----+---------+----------+-----------------+
++-----+---------+----------+-----------------+
+|  19 | Digital | I2C SCL  |                 |
++-----+---------+----------+-----------------+
+|  20 | Digital | I2C SDA  |                 |
```

Comment:
These are still I2C on V2 as well, even if it's a different bus.

Reply:
Fixed in https://github.com/bbcmicrobit/micropython/pull/692/commits/78bd2418a5e2b22c538b233e3313806f2dc032e5